web/packages/teleterm/src/ui/Search/pickers/ActionPicker.tsx

```diff
@@ -608,15 +608,21 @@ export function ServerItem(props: SearchResultItem<SearchResultServer>) {
   );
 
   return (
-    <IconAndContent Icon={icons.Server} iconColor="brand">
+    <IconAndContent
+      Icon={icons.Server}
+      iconColor="brand"
+      iconOpacity={getRequestableResourceIconOpacity(props.searchResult)}
+    >
       <Flex
         justifyContent="space-between"
         alignItems="center"
         flexWrap="wrap"
         gap={1}
       >
         <Text typography="body1">
-          Connect over SSH to{' '}
+          {props.searchResult.requiresRequest
+            ? 'Request access to server '
```

Comment:
In the unified view this works slightly different: if you already have a resource in the access request, all buttons (even on accessible resources) change to 'Request access'. This is because when you assume a resource request, you lose access to other things that aren't in the request. 

However, in the search bar I only show 'Request access' for the resources that the user doesn't have access to. What are your thoughts on this? Should the search bar switch to the 'request resources' mode too (for a given workspace)? cc @avatus

Reply:
One argument against switching the search bar to the request resources mode would be that access requests are scoped per root cluster. So the user would end up with some resources being in "request access" mode and others in "connect" mode.

This is not necessarily bad and having multiple root clusters is rare. Nonetheless, it's one of the use cases we built the search bar for.

I don't have a strong feeling on this, but as you said in the PR description, I think we should keep the search bar as simple as possible. This is the same reason for which for TCP apps you cannot _either_ use VNet or a local proxy from the search bar. You can only select one action which is the preferred one.

Reply:
I agree with both of your sentiments here. IMO, no one is going to be "building" an access request through the search bar, but only starting one. The rest is most likely done in the document. (of course, this is an assumption). I think the search bar as simple as possible is 👍

Reply:
Thanks for the feedback guys, looks like we all agree on keeping the search bar simple!